website/blog/2019-12-09-beta-release.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Announcing Beta of component-test-utils
+author: Florent Berthelot
+authorURL: http://twitter.com/berthel350
+---
+
+I've been working on this side-project for almost 1 year now and I think it's mature enough to pass another step: The Beta step !
```

Comment:
`step!`

Reply:
Pas fan de la repetition de step. Pourquoi pas:

`...I think it's mature enough to go onto the next step: the official Beta!`